backend/src/main/java/trn/logistics/knapsack/rest/MaterialController.java

```diff
@@ -0,0 +1,62 @@
+package trn.logistics.knapsack.rest;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import trn.logistics.knapsack.dto.Material;
+import trn.logistics.knapsack.service.MaterialService;
+
+import java.util.List;
+
+import static trn.logistics.knapsack.rest.MaterialController.API;
+
+@Slf4j
+@RestController
+@RequestMapping(API)
+public class MaterialController {
+
+    public static final String API = "/api/v1/materials";
+    
+    @Autowired
+    private MaterialService materialService;
+
+    @CrossOrigin
+    @PutMapping
+    public void updateMaterial(@RequestBody Material material) {
+        materialService.addMaterial(material);
+    }
+
+    @CrossOrigin
+    @PostMapping(consumes = "application/json", produces = "application/json")
```

Comment:
```suggestion
    @PostMapping()
```
This should be enough